Find404.scala

```diff
@@ -0,0 +1,61 @@
+import java.io._
+import scala.io.Source._
+import scala.io.StdIn._
+import scala.collection.mutable.ArrayBuffer
+
+var fileSet = ArrayBuffer[File]()
```

Comment:
Move all this stuff to a [main method](https://www.scala-lang.org/documentation/your-first-lines-of-scala.html)

Reply:
Global variables tend to cause problems, moving this stuff to a main method addresses some of those problems as the scope of these variables are limited to that function. Doing this makes it easier to reason about what is going on in the code.